docs/src/content/docs/guides/uploading-images-and-videos.mdx

```diff
@@ -1,11 +1,113 @@
 ---
 title: Uploading Images & Videos
-description: Uploading Images & Videos
+description: Learn how to upload images and videos in Astro with Cloudinary
 head:
   - tag: title
     content: Uploading Images & Videos - Astro Cloudinary
 ---
-import { CldImage } from '../../../../../astro-cloudinary';
+import { CldUploadWidget } from '../../../../../astro-cloudinary';
 
 import CodeBlock from '../../../components/CodeBlock.astro';
-import HeaderImage from '../../../components/HeaderImage.astro';
\ No newline at end of file
+import HeaderImage from '../../../components/HeaderImage.astro';
+import Button from '../../../components/Button.astro';
+import DemoUploadWidgetSigned from '../../../components/DemoUploadWidgetSigned.astro';
+
+
+Astro Cloudinary provides a seamless way to integrate Cloudinary's powerful upload capabilities into your Astro project. This guide will show you how to use the `CldUploadWidget` component to add upload functionality to your application.
+
+## CldUploadWidget Component
+
+The `CldUploadWidget` component is a wrapper around Cloudinary's Upload Widget, specifically optimized for Astro projects. It allows you to easily add upload functionality with minimal configuration.
+
+### Basic Usage
+
+Here's a simple example of how to use the `CldUploadWidget`:
+
+<CodeBlock>
+```astro copy
+
+import { CldUploadWidget } from 'astro-cloudinary';
+
+<CldUploadWidget uploadPreset="your_upload_preset">
+  <Button>Upload to Cloudinary</Button>
```

Comment:
can you make this `<button>`? it implies here there's an existing Button component, so would rather show an example with the native HTML element